### PR TITLE
Add US CA Tulare Scaper

### PR DESCRIPTION
**Pull request by go717franciswang**
Ran unit test, lint check, type check, and E2E test.

### CODE REVIEW

File: recidiviz/tests/ingest/scrape/regions/us_ca_tulare/us_ca_tulare_scraper_test.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        # Tests scraping data. Fill in |content| with the state of the page
+        # containing person data, and |expected_result| with the IngestInfo
+        # objects that should be scraped from the page. A default Task will be
+        # passed to the |populate_data| method, but this can be overriden by
+        # supplying a |task| argument
+        content = _PERSON_HTML
+        expected_info = IngestInfo(
+                people=[Person(
+                    person_id='6191',
+                    full_name='BRAVO, FLORENTINO HIPOLITO',
```
Comment (jamwalla) [inline recidiviz/tests/ingest/scrape/regions/us_ca_tulare/us_ca_tulare_scraper_test.py]: can you scrub out the name, birthdate and ID here and in the html? The roster page should also not include real names/dobs

File: recidiviz/ingest/scrape/regions/us_ca_tulare/us_ca_tulare_scraper.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            endpoint = 'http://tularecounty.ca.gov' + link.get('href')
+            task_list.append(Task(
+                task_type=constants.TaskType.SCRAPE_DATA,
+                endpoint=endpoint))
+        return task_list
+
+    def get_enum_overrides(self) -> EnumOverrides:
+        overrides_builder = EnumOverrides.Builder()
+        overrides_builder.add('M', Race.OTHER)
+        overrides_builder.add('AR', Race.OTHER)
```
Comment (jamwalla) [inline recidiviz/ingest/scrape/regions/us_ca_tulare/us_ca_tulare_scraper.py]: Again on context, I think this is "Arab". The unfortunate leap here is to map this to `Race.WHITE`.

**Comment by go717franciswang** [comment]
Hi jamwalla@, thanks for the thorough review, and sorry about the delay. All issues addressed! PTAL.

**Comment by go717franciswang** [comment]
Great! I'm slowly getting the hang of this, and hopefully contribute more often.

**Pull request merged**